furusiya/model/maps/area_map.py

```diff
@@ -2,6 +2,15 @@
 
 
 class AreaMap:
+
+    # TODO: these should be the renderer's concern, not the map's.
```

Comment:
Or perhaps the map generator's?
Either way, one thing's for sure: they don't belong here.